extensions/qute/qute-reference-guide.adoc

```diff
@@ -0,0 +1,673 @@
+:toc:
+:toc-placement!:
```

Comment:
Why is it in the extensions directory? It won't be visible on the website, whereas it looks very valuable.

I think it should be placed alongside the cdi guide and be called `qute-reference` to be consistent with https://quarkus.io/guides/cdi-reference .

Reply:
Well, I wasn't sure that the docs could contain a "real" reference guide. The CDI ref guide is kind of special - it's not a real reference guide but back then we did not find a better name.